src/main/java/io/seqera/tower/cli/commands/pipelines/LaunchOptions.java

```diff
@@ -39,7 +39,7 @@ public class LaunchOptions {
     @Option(names = {"--revision"}, description = "A valid repository commit Id, tag or branch name.")
     public String revision;
 
-    @Option(names = {"--config"}, description = "Additional Nextflow config file.")
+    @Option(names = {"--config"}, description = "Path to a Nextflow config file. Values defined here override the same values in the pipeline repository config file.")
```

Comment:
I think it's not precise, or at least it seems to me that the override is key-based while it's not the case.
It is a total override, discarding completely what comes from the pipeline.